src/OSPSuite.Presentation/Presenters/Importer/ColumnMappingPresenter.cs

```diff
@@ -523,14 +533,15 @@ public IEnumerable<ColumnMappingOption> GetAvailableOptionsFor(ColumnMappingDTO
          return options;
       }
 
-      private IEnumerable<string> 
+      private IEnumerable<string>
          availableColumns()
       {
          return _format.ExcelColumnNames
             .Where
             (
                cn =>
-                  _format.Parameters.OfType<MappingDataFormatParameter>().All(p => p.ColumnName != cn && p.MappedColumn?.Unit?.ColumnName != cn && p.MappedColumn?.LloqColumn != cn) &&
+                  _format.Parameters.OfType<MappingDataFormatParameter>().All(p =>
```

Comment:
This statement needs to be rewritten at some point. I have absolutely NO idea what this means. So many ANDs, NOT, without comment. Let's be kind to own future selves and not do that anymore

Reply:
OK, I ll add a todo here for now and we can talk about refactoring such code.